guides/project-service-management-guide.md

```diff
@@ -251,6 +251,72 @@ application repositories
 **NOTE** `spk service` command documentation can be found
 [here](/guides/service-management.md).
 
+#### Helm Configuration for SPK
+
+`spk service create` allows a user to configure a service a number fo ways with
+a backing helm chart.
+
+Presently, there are are a number of options for `spk service create` documented
+below:
+
+```
+  -c, --helm-chart-chart <helm-chart>                         bedrock helm chart name. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+  -r, --helm-chart-repository <helm-repository>               bedrock helm chart repository. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+
+  -g, --helm-config-git <helm-git>                            bedrock helm chart configuration git repository. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+  -b, --helm-config-branch <helm-branch>                      bedrock custom helm chart configuration branch. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+  -p, --helm-config-path <helm-path>                          bedrock custom helm chart configuration path. --helm-chart-* and --helm-config-* are exclusive; you may only use one. (default: "")
+```
+
+As noted by the the documentation text, `helm-chart-*` and `helm-config-*` are
+both mutually exclusive configurations ie: you can _only_ use one set of
+configurations or the other.
+
+This section intends on documenting the various use cases for both sets of
+mutually exclusive configurations.
+
+##### Helm Charts in a well-known Helm Repository
+
+A Helm Repository is a well known set of helm charts conforming to the
+[helm repository guidelines](https://helm.sh/docs/topics/chart_repository/).
+Perhaps the best known helm repository is the community run
+[helm charts repository](https://github.com/helm/charts).
+
+As an `spk` user, if you would like to incorporate helm charts from a well known
+public repository, you may simply run `spk` the following `helm-chart`
+arguments:
+
+```
+spk service create --helm-chart-chart stable/nginx --helm-chart-repository github.com/helm/charts
+```
+
+##### Helm Charts in a distinct Git Repository
```

Comment:
Do we want to include a blurb here about PATs and access?

Reply:
Agree with @mtarng

Reply:
fixed